test/e2e/framework/gpu_util.go

```diff
@@ -32,7 +32,7 @@ const (
 	// TODO: Parametrize it by making it a feature in TestFramework.
 	// so we can override the daemonset in other setups (non COS).
 	// GPUDevicePluginDSYAML is the official Google Device Plugin Daemonset NVIDIA GPU manifest for GKE
-	GPUDevicePluginDSYAML = "https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml"
```

Comment:
I think we want to test both stable and master branches. `stable` is what we want users to use, but we want to make changes first to `master` branch.

Reply:
This is in 1.9 release branch. On head, we are testing against master.